packages/go/constants/constants.go

```diff
@@ -0,0 +1,78 @@
+package constants
```

Comment:
nit: `constants` is not a great package name. How about having a top level `osv` package for things like these?

Reply:
Given how Go naming package conventions work, I think it's actually a pretty good name given what it's trying to avoid - specifically, trying to avoid dependency cycles by having a package which should never have logic in it to avoid ever needing to do an import itself.

Having said that, `osvcontants` might be slightly better to reduce the likelihood of namespace clashes (which is the issue with naming it `osv`) 🤔

Reply:
How about `osvschema` ? We would keep it as a place for defining structs and constants, but it would have a slightly more descriptive name.

Reply:
@another-rex

Reply:
Sure, just so long as it's clear when you say "structs", that the idea is there should never be any logic.

The whole idea around this was it'd be a constant-only package to ensure there will never be any cycles - while in theory it can have structs too, imo that makes it very tempting to have logic. We can try to avoid that but it's required for any special handling of marshalling, and then that makes it tempting to have _just a bit more logic_, and then we're back where we are today with cyclical dependencies...

That's why we decided we'd have this package dedicated to just constants, and have all the structs in another package

Reply:
By struct, I mean any structs for e.g. defining the OSV schema structures. 

As a whole, I don't think any bindings code in this repository should have much logic at all on top of just defining constants and structs. It should just contain a bunch of definitions + some very lightweight helpers to help others use the OSV format. 

We should keep things to a bare minimum, and maintain more complicated logic elsewhere.

Reply:
happy to go with `osvschema`

Reply:
done